erpnext/buying/doctype/request_for_quotation/request_for_quotation.py

```diff
@@ -226,3 +227,16 @@ def create_rfq_items(sq_doc, supplier, data):
 		"request_for_quotation_item": data.name,
 		"request_for_quotation": data.parent
 	})
+
+@frappe.whitelist()
+def download_pdf(doctype, name, supplier_idx):
+	doc = get_rfq_doc(doctype, name, supplier_idx)
+	if doc:
+		frappe.get_attr("frappe.www.print.download_pdf")(doctype, name, doc=doc)
```

Comment:
from frappe.www.print import download_pdf
I have tried this, but it giving me an error invalid syntax at print
Seems like it's not allowing to import with filename print